Dokumentationen/test/test_cases.adoc

```diff
@@ -317,7 +298,7 @@ a| * gefilterte, inkl. aller gebuchten Experimente, Dozentenwoche
 
 |*Beschreibung* |
 
-|*Vorbedingungen und benötigte Daten*
+|*Vorbedingungen und benötigte Daten und benötigte Daten*
```

Comment:
Ich vermute mal ein Kopierfehler oder? :D Ist auch bei allen folgenden TCs falsch
```suggestion
|*Vorbedingungen und benötigte Daten*
```

Reply:
Super 🙃 Ich hoffe, dass ihr das auch gesehen und korrigiert habt. Ansonsten mache ich das nochmal ganz zum Schluss.

Reply:
Du hast es ja hier erst hinzugefügt, also haben wir das in unseren Branches noch nicht und können es dementsprechend nicht entfernen